sql/migrations/user-collection/up.sql

```diff
@@ -0,0 +1,30 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS bookbrainz.user_collection (
+	id UUID PRIMARY KEY DEFAULT public.uuid_generate_v4(),
+	editor_id INT,
+	name VARCHAR(80) NOT NULL CHECK (name <> ''),
+	description TEXT NOT NULL DEFAULT '',
+	type bookbrainz.entity_type NOT NULL,
```

Comment:
What do you think about `entity_type` instead of `type`?

Reply:
I prefer `type` here. `entity_type` would be fine too. Okay with either of them :P what do you say? should I change it to `entity_type`

Reply:
I think entity_type is clearer. Otherwise it suggests this column is "collection type" which could be interpreted different ways (public/private, for example?).
It's a bit of a nitpick though and both would work.

Reply:
yeah true, I didn't think of public/private. I'll change it :)